dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -312,9 +312,15 @@ function disk_has_unallocated_space {
     local pt_table_type
     pt_table_type=$(get_partition_table_type "${disk_device}")
     if [ "${pt_table_type}" = "dos" ];then
-        sfdisk --verify "${disk_device}" 2>&1 | grep -q "unallocated"
+        # we can't distinguish from 'intentional free space left'
+        # and the 'already resized' condition. Thus assume it's
+        # not fully allocated to allow for resize
+        true
+    elif [ "${pt_table_type}" = "gpt" ];then
+        sgdisk --verify "${disk_device}" 2>&1 grep -q "end of the disk"
```

Comment:
Pipe missing :-)

Reply:
thanks, this code doesn't like me... or maybe the virus

Reply:
wonder why shellcheck did not complain, probably it did not expect that stupid mistakes ;)